dashboard/agent.py

```diff
@@ -62,9 +62,13 @@ def __init__(self,
         self.object_store_name = object_store_name
         self.raylet_name = raylet_name
         self.node_id = os.environ["RAY_NODE_ID"]
-        self.ppid = int(os.environ["RAY_RAYLET_PID"])
-        assert self.ppid > 0
-        logger.info("Parent pid is %s", self.ppid)
+        # TODO(edoakes): RAY_RAYLET_PID isn't properly set on Windows. This is
+        # only used for fate-sharing with the raylet and we need a different
+        # fate-sharing mechanism for Windows anyways.
+        if sys.platform != "win32":
```

Comment:
Do you know if the code path will work when `sys.platform == "cygwin"`? I'm not sure the behavior here, but I think it is worth checking that the platform is also not `cygwin`.
https://docs.python.org/3/library/sys.html#sys.platform

Reply:
good idea